src/bootstrap/tool.rs

```diff
@@ -304,6 +304,12 @@ impl Step for Rustdoc {
                                            target,
                                            "build",
                                            "src/tools/rustdoc");
+
+        // Most tools don't get debuginfo, but rustdoc should.
+        cargo.env("RUSTC_DEBUGINFO", builder.config.rust_debuginfo.to_string())
+             .env("RUSTC_DEBUGINFO_LINES", builder.config.rust_debuginfo_lines.to_string())
+             .env("RUSTC_FORCE_UNSTABLE", "1");
```

Comment:
I don't think this env var is necessary because we're not shipping rlibs, just the rustdoc executable